MAUI/DataGrid/grid-events.md

```diff
@@ -152,6 +218,19 @@ private void dataGrid_CellHovered(object sender, DataGridCellHoveredEventArgs e)
 {% endhighlight %}
 {% endtabs %}
 
+### CellHovered Command and its Parameter
+When the cell is hovered, the `CellHoveredCommand` binds to the command which you defined in your ViewModel. Additionally, we have the option to utilize the `CellHoveredCommandParameter` for passing data to the Command upon its execution.
```

Comment:
The command will be invoked when the mouse pointer hovers over a cell in the SfDataGrid. The instance of `DataGridCellHoveredEventArgs` will be passed as the default value of the command's parameter. It can be customized by setting the `CellHoveredCommandParameter` property.

Reply:
Changed